inventory/sample/group_vars/all/all.yml

```diff
@@ -35,6 +35,10 @@ loadbalancer_apiserver_healthcheck_port: 8081
 
 ### OTHER OPTIONAL VARIABLES
 
+# If true, doesn't include host nameservers in the nameserverentries
+# If enabled, consider defining your DNS servers. Otherwise, there may be problems with dns queries.
```

Comment:
s/If enabled/If false/ to understand the meaning easily by making it consistent with the above line.

Reply:
Thank you for the review. I just updated the comment.